src/DIRAC/WorkloadManagementSystem/Agent/PilotLoggingAgent.py

```diff
@@ -0,0 +1,89 @@
+""" :mod: PilotLoggingAgent
+
+    PilotLoggingAgent sends Pilot log files to an SE
+"""
+
+# # imports
+import os, requests
```

Comment:
```suggestion
import os
import requests
```

Reply:
OK, you were quicker than me here. Yjere is n client now in place, not a bare requests call.

Reply:
I followed @andresailer advice and used a `@executeWithUserProxy` decorator approach. It works as advertised.